Task2/Kuba/NoughtsAndCrossesWPF/NoughtsAndCrossesWPF/Model/Game.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace NoughtsAndCrossesWPF
+{
+    class Game
+    {
+        public IBoardCommunication interpreter;
+        GameStatus gameStatus;
+        public Game(IBoardCommunication interpreter)
+        {
+            this.interpreter = interpreter;
+        }
+
+        public void GameActions(Player currentPlayer, GameBoard board, object field)
+        {
+            MakeMove(board, currentPlayer, field);
+            interpreter.ShowBoard(board);
```

Comment:
whenever you have verb 'Show' in a Model it's a good sign it should be in a ViewModel